lib/invoker/power/setup/common.rb

```diff
@@ -0,0 +1,17 @@
+require 'invoker/power/power'
+
+module Invoker
+  module Power
+    class Setup
+      module Common
+        def tld
+          Invoker::Power.tld
+        end
+
+        def tld_value
+          Invoker::Power.tld_value
+        end
+      end
+    end
+  end
+end
```

Comment:
This is an old file from a work in progress commit that should have been removed. Doing it now.